Cargo.toml

```diff
@@ -21,7 +21,9 @@ gumdrop = "0.7"
 hkd32 = { version = "0.3", default-features = false, features = ["mnemonic"] }
 hkdf = "0.8"
 hmac = "0.7"
+merlin = "2.0.0"
 once_cell = "1.3"
+orion = { version = "0.15.1", default-features = false }
```

Comment:
It doesn't look like you're using this for anything? Also the primitives it provides should already be available

Reply:
yes, I removed it.